yarn.lock

```diff
@@ -465,6 +465,37 @@
     tslib "^1.8.0"
     uuid "^3.2.1"
 
+"@aws-amplify/analytics@6.0.24":
```

Comment:
Think these yarn.lock updates are extraneous, can we check out the _yarn.lock_ from `main` and rerun `yarn` to prevent these?

Reply:
doing that right now